actions/push-to-gar-docker/action.yaml

```
@@ -170,3 +170,9 @@ runs:
         platforms: ${{ inputs.platforms }}
         ssh: ${{ inputs.ssh }}
         build-contexts: ${{ inputs.build-contexts }}
+
+    - name: Cleanup checkout directory
+      if: always()
```

```suggestion
      if: ${{ !cancelled() }}
```

From [here](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions#always). `always()` has caused us problems with stuck jobs before - it gets in the way of cancellation...